namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-community-accommodation-test/resources/irsa.tf

```
@@ -23,7 +23,12 @@ module "irsa" {
   eks_cluster_name       = var.eks_cluster_name
   service_account_name   = "hmpps-community-accommodation-api-service-account"
   namespace              = var.namespace
-  role_policy_arns       = local.sns_policies
+  role_policy_arns       = merge(
+    {
+      rds = module.cas-2-domain-events-queue.irsa_policy_arn
```

should remove reference to rds.  The key here is not really used - other than to help identify the SQS queue policy.  So you could have:

```suggestion
      cas-2 = module.cas-2-domain-events-queue.irsa_policy_arn
```